core/migrate/data.go

```diff
@@ -97,9 +97,9 @@ var migrations = []migration{
 		ALTER TABLE config ALTER COLUMN blockchain_id SET DATA TYPE bytea USING decode(blockchain_id, 'hex');
 	`},
 	{Name: "2017-01-13.0.core.asset-definition-bytea.sql", SQL: `
-		ALTER TABLE assets 
+		ALTER TABLE assets
 			ALTER COLUMN definition SET DATA TYPE text;
-		ALTER TABLE assets 
+		ALTER TABLE assets
```

Comment:
(resolved offline, but for those following along at home.... we'll preserve this for now but plan on collapsing into a single migration before we do the next blockchain reset.)